data/yam/agama/auto/autoyast_supported_ssh_pub_key.xml

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
For AutoYaST we don't plan to have anything like what we do with jsonnet to create them dynamically so we should minimize the number of files, could you please edit existing file and use it for all architectures

Reply:
Updated. In fact, originally I added this in the existing file. :)